ci-operator/jobs/openshift/osde2e/openshift-osde2e-main-periodics.yaml

```diff
@@ -2397,6 +2397,75 @@ periodics:
     - name: osde2e-common
       secret:
         secretName: osde2e-common
+- agent: kubernetes
+  cluster: build03
+  cron: 0 */12 * * *
+  decorate: true
+  extra_refs:
+    - base_ref: main
+      org: openshift
+      repo: osde2e
+  labels:
+    pj-rehearse.openshift.io/can-be-rehearsed: "false"
+  name: osde2e-stage-aws-addon-ocs-consumer-qe
+  reporter_config:
+    slack:
+      channel: '#forum-ocs-dedicated'
+      job_states_to_report:
+        - failure
+      report_template: Job {{.Spec.Job}} failed.
+  spec:
+    containers:
+      - args:
+          - test
+          - --secret-locations
+          - $(SECRET_LOCATIONS)
+          - --configs
+          - $(CONFIGS)
+        command:
+          - /osde2e
+        env:
+          - name: ADDON_IDS
+            value: ocs-consumer-qe
+          - name: ADDON_PARAMETERS
+            value: '{"ocs-consumer-qe":{"size": "1", "storage-provider-endpoint": "10.0.20.236:31659", "onboarding-ticket": "eyJpZCI6Ijc3N2E4ZmE5LTYyM2UtNGNhMS1hM2NmLTNmNjk4MDM2MGUwNSIsImV4cGlyYXRpb25EYXRlIjoiMjY0NjI5ODcxNyJ9.rjiFpGvZZMkLbJEeKt1cSuVBmBOwl0yxgXo9Na5W58gdjdxT7H7QsgF1VFf/HT95h+pDHBlG5DCu8EAkBIKLkyRR0tXZIW3qo4rbfNowh2QqvQlXDISorRX9kct0B4VC1he0YIPVHEHxUnUfExanBVoFus2zJj/0WRS8WSbSRhM0EoJS+F1tG1mxSwqBnzyRAy5ki36CPLq5TH4YU826u64Tv1/4NZuppEIGvnZpcTlh+66wrH2X9edAJpovps/fA44zeycpkA9HsKLXTaUziqbRWC54BZbyLz9LBEqA2N5F3ha1ckrRqblepdiTHxwuEcVENr7iUleLs30JEAHDywCHT8YKmajrTaWYyIUEEcZ6BOLald4qshc9XAVkLbbB+zTaFk3UD5/OJtgqW4LDEbtFAarT/G5SC1dwDIiYlOXhB/OCQ7/tjbAdHu4EUqIfUWJsCgb7KOJm1myYIHUYwIiBK9kRHhV4YI8WkLMziaRhWIMmV+Uzj9jM5yAn696CqPLd/uauvzuuT252vEq1D7zKtlJHhJ/ww6/8ob7tF6s1k4bHbaRAUY9a6wL8ipS4LyrJbmtbCF9FQP3/AYoeSp4w0VzhE5AwgsETcE3GujhBXPMaOZEWGXyf+1jsXUsPDNO5Q160NgMVPKCr0fSJrR9kmZGxUrZINuPi7gI+ddw="}}'
```

Comment:
I think your onboarding ticket will get invalidated after the first run as it will be marked as used.

Reply:
Addressed.